src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -1410,3 +1410,124 @@ def _reorder_cache(self, past, beam_idx) -> Tuple:
 
             reordered_decoder_past = reordered_decoder_past + (reordered_layer_past_states,)
         return past + (reordered_decoder_past,)
+
```

Comment:
same changes should be done for TF

Reply:
Done. Hopefully, I didn't miss anything.